Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/ListViewExtensions.cs

```diff
@@ -33,6 +35,11 @@ public static class ListViewExtensions
         /// </summary>
         public static readonly DependencyProperty StretchItemContainerDirectionProperty = DependencyProperty.RegisterAttached("StretchItemContainerDirection", typeof(StretchDirection), typeof(ListViewExtensions), new PropertyMetadata(null, OnStretchItemContainerDirectionPropertyChanged));
 
+        /// <summary>
+        /// Attached <see cref="DependencyProperty"/> for binding an <see cref="System.Windows.Input.ICommand"/> to handle ListViewBase Item interaction by means of <see cref="Windows.UI.Xaml.Controls.ListViewBase"/> ItemClick event. ListViewBase IsItemClickEnabled must be set to true.
+        /// </summary>
+        public static readonly DependencyProperty CommandProperty = DependencyProperty.RegisterAttached("Command", typeof(ICommand), typeof(ListViewExtensions), new PropertyMetadata(null, OnCommandPropertyChanged));
```

Comment:
We should have a `CommandParameter` as well, right?

Reply:
We wouldn't expose a CommandParameter attached property because the parameter is always the ListView's SelectedItem. Unless Ive missed something ?